### ISSUE TITLE
[Bug][a11y]Common myth hyperlink button to small - consider making larger

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#7887]
### Prerequisites
1. Make sure you have Google Accessibility Scanner installed.

## Steps to reproduce
1. Tap on the private mode button;
2. Without any page opened, scan the "private mode" section using the Accessibility Scanner.

### Expected behavior
There are no suggestions to make any changes.

### Actual behavior
There is a suggestion to consider making the Common myth hyperlink button larger - as seen in the screenshot below.

### Device information

* Android device: Samsung Galaxy Tab S3 (Android 8.0)
* Fenix version: 3.2.0-beta.2 #20230233 
![Screenshot_20200123-160701_Accessibility Scanner](https://user-images.githubusercontent.com/40264165/72991386-950d0c00-3dfa-11ea-9dc3-21405d46df4f.jpg)

**Comment by Diana-Rus** [comment]
Hi, this ticket has a duplicate #7708. Because here the priority was set i am going to close the other one.
Put as note the comment from the other:
[sblatz commented on Jan 29](https://github.com/mozilla-mobile/fenix/issues/7708#issuecomment-579696046)

**Comment by mcarare** [comment]
I am filling a separate issue for the type announcement as it is a problem that should be addressed for multiple similar links: #10285

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **Nightly Build 200512 from 5/12** using the following devices:
• Google Pixel 3a (Android 10)
• OnePlus A3 (Android 6.0.1)
